Automattic/jetpack
# Sharing: remove Publicize section in WP Admin > Settings > Sharing

This PR brings parity between the Publicize settings in WP Admin in wpcom and jp sites.

#### Changes proposed in this Pull Request:


* remove Publicize section in WP Admin > Settings > Sharing

<img width="997" alt="captura de pantalla 2019-02-08 a la s 18 48 54" src="https://user-images.githubusercontent.com/1041600/52510928-bd492e00-2bdc-11e9-9fe1-a1a95ed45802.png">

#### Testing instructions:





* Ensure Jetpack is connected and Publicize is active
* go to WP Admin > Settings > Sharing
* the Publicize section should look like the one above

#### Proposed changelog entry for your changes:


* The Publicize section in WP Admin > Settings > Sharing was removed. The section to configure sharing buttons is still available. Users can continue configuring Publicize connections in the dedicated page in WordPress.com.
